pkg/apis/pingcap/v1alpha1/validation/validation.go

```diff
@@ -168,6 +171,15 @@ func validateComponentSpec(spec *v1alpha1.ComponentSpec, fldPath *field.Path) fi
 	return allErrs
 }
 
+//validateRequestsStorage validates resources requests storage
+func validateRequestsStorage(requests corev1.ResourceList, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	if _, ok := requests[corev1.ResourceStorage]; !ok {
+		allErrs = append(allErrs, field.Required(fldPath.Child("requests.storage").Key((string(corev1.ResourceStorage))), "PD or TiKV request storage must not be empty"))
```

Comment:
```suggestion
		allErrs = append(allErrs, field.Required(fldPath.Child("requests.storage").Key((string(corev1.ResourceStorage))), "storage request must not be empty"))
```

We may invoke this method to verify other components and the error will include the field ref which is self-explanatory